lib/util.js

```diff
@@ -346,7 +346,8 @@ function formatValue(ctx, value, recurseTimes) {
   // Provide a hook for user-specified inspect functions.
   // Check that value is an object with an inspect function on it
   if (ctx.customInspect && value) {
-    const maybeCustomInspect = value[customInspectSymbol] || value.inspect;
+    const maybeCustomInspect = value[customInspectSymbol] ||
```

Comment:
If that's the case maybe we need to add a comment here about why we are more careful about normal properties than symbols..